### PR TITLE
[No Review][No Merge][WIP][HOLD] POC for `CompleteGuidedSetup`

### PR DESCRIPTION
**Pull request by francoisl**
### Details
This is just an example for using a new API command that we won't merge, no need to review.

### CODE REVIEW

File: src/libs/actions/PersonalDetails.ts [hunk 1]
```diff
@@ -276,9 +281,172 @@ function openPublicProfilePage(accountID: number) {
...
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${parentReportID}`,
+            value: {
+                // Message 1 (video)
+                [reportCommentAction.reportActionID ?? '']: reportCommentAction as ReportAction,
+                // Message 2 (plain text)
+                [reportCommentAction2.reportActionID ?? '']: reportCommentAction2 as ReportAction,
+            },
+        }, {
+            // This doesn't seem to work, not sure why
```
Comment (mountiny) [inline src/libs/actions/PersonalDetails.ts:429]: could you just use merge? sometime using mix of set and merge behaves werid

Comment (NikkiWines) [reply src/libs/actions/PersonalDetails.ts:429]: Vit! 😂 
image

Comment (mountiny) [reply src/libs/actions/PersonalDetails.ts:429]: I have just added a single comment, not a review 😂 idk why Github is this way

**Comment by francoisl** [comment]
We no longer need this, closing.

**Pull request closed without merging**